<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservation</class>
 <widget class="QDialog" name="reservation">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>663</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Reservation</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Consulter Reservation</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/logo.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_search">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criteres de recherche :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_recherche">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#bt_recherche{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_recherche:hover{

background-color:#27a9e3;

}</string>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ID reservation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID personne</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prenom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID livre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom de livre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date reservation</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="table_reservation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>1231</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID RESERVATION</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID PERSONNE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOM PERSONNE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PRENOM PERSONNE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID LIVRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TITRE LIVRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATE RESERVATION</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bt_reload">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>110</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#bt_reload{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_reload:hover{

background-color:#27a9e3;

}</string>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
